src/online/mongo_communications.go

```diff
@@ -1,7 +1,47 @@
 package main
 
-import "fmt"
+import (
+	"bytes"
+	"encoding/json"
+	"net/http"
+	"vaas/structs"
+)
 
-func mongo() {
-	fmt.Println("MONGO COMMUNCIATIONS RUNNING")
+func mongo_createSharedGame(sharedGame structs.SharedGame) (string, error) {
```

Comment:
This function never actually requires us to return a string, so I'd recommend making this change and the changes I've added throughout this function.
```suggestion
func mongo_createSharedGame(sharedGame structs.SharedGame) error {
```